src/components/biggive-header/biggive-header.scss

```
@@ -205,6 +205,13 @@
       background-repeat: no-repeat;
       padding-left: 25px;
     }
+    li.icon-colour-brand-mhf-turquoise a {
+      background-image: url('/assets/images/biggive-double-triangle-brand-mhf-turquoise.svg');
+      background-position: left center;
+      background-size: 20px auto;
+      background-repeat: no-repeat;
+      padding-left: 25px;
+    }
```

This is not needed right now - we're not putting MHF in the header today, and we don't actually have a turquoise triangle asset.
```suggestion
    li.icon-colour-brand-mhf-turquoise a {
      background-image: url('/assets/images/biggive-double-triangle-brand-mhf-turquoise.svg');
      background-position: left center;
      background-size: 20px auto;
      background-repeat: no-repeat;
      padding-left: 25px;
    }
```